src/components/OrganizationSwitch.tsx

```diff
@@ -99,16 +99,16 @@ const OrganizationSwitch = () => {
           <span>Select Organization</span>
           <ul>
             <li>
-              <Link to="/">username</Link>
+              <Link to="/">{state.user.username ?? "unknown"}</Link>
             </li>
-            {state.fakeOrganizations.map((organization) => (
+            {state.user.orgs?.map((organization) => (
               <li
                 className={
                   location.pathname === "/" + organization.name
                     ? "selected-organization"
                     : ""
                 }
-                key={organization.publicKey}
+                key={organization.slug}
```

Comment:
slug is useful here because it should be unique, and isn't as long as a public key